plugins/legacy-preset-chart-nvd3/src/NVD3Vis.js

```diff
@@ -422,6 +429,16 @@ function nvd3Vis(element, props) {
           const total = d3.sum(data, d => d.y);
           chart.tooltip.valueFormatter(d => `${((d / total) * 100).toFixed()}%`);
           chart.labelType(d => `${d.data.x}: ${((d.data.y / total) * 100).toFixed()}%`);
+        } else if (pieLabelType === 'key_value_percent') {
```

Comment:
It seems like we could DRY this up, there's quite a bit of copy/paste here.

Reply:
done

Reply:
done